src/app/static/src/app/components/ChoroplethFilters.vue

```diff
@@ -143,6 +177,10 @@
                 if (newSurveyFilter) {
                     this.selectSurvey(newSurveyFilter);
                 }
+
+                if (!this.selectedChoroplethFilters.region && this.regionOptions) {
```

Comment:
It's actually coming back from the API as always having a single top-level area object ("Malawi") with everything else nested underneath. We'll always have a selected region filter, which will default to this top-level area. 
I agree that this means 'regionOptions' is a confusing name though, since it's singular rather than plural. 'regionTree' or 'nestedRegions'?

Reply:
Should this maybe be changed in hintr then, to return an array as per the other filters? The generated types have it as an array, so perhaps this is a regression? https://github.com/mrc-ide/hint/blob/master/src/app/static/src/app/generated.d.ts#L186

Reply:
I think it makes sense as a tree structure as we're expecting a single nested country structure, and it's not completely analogous to the other filters since it is nested. So maybe rather than making an array to make it  conform with the other filters, we should rename it to make clear that it's a bit different? Unless we think we might really have to deal with separate region trees? I think the generated types are out of date.

Reply:
I guess it just seems like if you're wrapping it in an array anyway to use it, why not get it back in that form? 
https://github.com/mrc-ide/hint/pull/71/files#diff-af2c04cc2dda9f0a2866952490093c68R104

Reply:
But if you think it's more useful as an object then yeah, lets regenerate types and name this object differently.

Reply:
I think it does mean the code is slightly simpler, although it does look a bit weird because of the discrepancy with the other (array) option types.